### PR TITLE
[Documentation / README] Simplified installation guide

### PR DESCRIPTION
**Pull request by bpolaszek**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | none |
| License         | MIT |

Hello there,

I suggest some minor changes in the README. When looking at the installation part, I found some confusing points:
* The guide assumes you've never used composer but you're comfortable with yarn (it will probably be the contrary)
* There's no "production" point.
* The code block looks big, and some may think the installation is tedious at first glance
* There are too many links to my eyes :)

I also changed the order of the sentences in the intro to conclude with "Enjoy being an eCommerce Developer again!" instead of having it somewhere in the middle.

What's your opinion?

### CODE REVIEW

**Comment by bpolaszek** [comment]
Please don't merge now, The _production_ part is not ok.

**Comment by bpolaszek** [comment]
Please don't merge now, the _production_ part is not ok.
Since we're on stable version we should prove that setting up a shop on production with Sylius is a piece of cake.

But actually when running
```bash
php bin/console sylius:install --env prod
```

The installer loads sample data in the database. I don't think this is relevant in a production environment.
So far I did not see an option in the `sylius:install` command that allows skipping fixtures.
There's also no intuitive & safe way to destroy sample data in a simple command line.

In my opinion:
* We should start by resolving #8476 - I think it's very confusing to force the developer to have 2 different databases. 
* We need:
    * Either an option to skip fixtures when calling `sylius:install`, either by detecting `--env prod` or by a new `InputOption` like `--skip-fixtures` (I prefer the 2nd option)
    * Or a new command `sylius:remove:sample-data` that will truncate all relevant tables.

To my eyes, the simplest installation guide should look like this:

-----------
Installation
------------

You need [composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) to install PHP packages and [yarn](https://yarnpkg.com/lang/en/docs/install/) to install JS packages.

```bash
$ composer create-project sylius/sylius-standard my-sylius-shop && cd my-sylius-shop
$ php bin/console sylius:install
$ yarn install && yarn run gulp
$ php bin/console server:start
```
_Alternatively, you can use [Vagrant](http://docs.sylius.org/en/latest/book/installation/vagrant_installation.html) for your initial setup._

Then:
* Open `http://localhost:8000/` in your web browser to enjoy your Sylius shop in a development environment. 
* Open `http://localhost:8000/admin` to discover your new back-office.

To help you get familiar with the Sylius interface, we populated sample products, taxons, orders, etc.

**Production**

When you're ready to go live, reset your database:

```bash
php bin/console sylius:remove-sample-data
```

Then please refer to [Symfony's documentation](https://symfony.com/doc/current/setup/web_server_configuration.html) to properly setup URL rewriting to `web/app.php` on  your Apache/Nginx web server.

-----------

What's your opinion? I think the installation part should target experienced Symfony developers as well as people that have very basic knowledge of PHP and want to quickly setup a fully functionnal online shop.

**Comment by bpolaszek** [comment]
Hi @CoderMaggie,

F*ck, you're right! 
```bash
Loading sample data for environment dev.
Warning! This action will erase your database.
Continue? (y/N)
```

I read too fast, actually, I thought I did not have a choice. When I read this, I understood that if I choose N, the whole installation process stops, so I systematically chose Y or Ctrl+C. My bad.

**Comment by bpolaszek** [comment]
Now, it's good for review :)

File: README.md [hunk 1]
```diff
@@ -37,20 +38,31 @@ Documentation is available at [docs.sylius.org](http://docs.sylius.org).
 Installation
 ------------
 
+You need [composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) to install PHP packages and [yarn](https://yarnpkg.com/lang/en/docs/install/) to install JS packages.
+
 ```bash
-$ wget http://getcomposer.org/composer.phar
-$ php composer.phar create-project sylius/sylius-standard project
-$ cd project
+$ composer create-project sylius/sylius-standard my-sylius-shop && cd my-sylius-shop
```
Comment (CoderMaggie) [inline README.md]: `MySyliusShop`?
Or even just `acme`?

**Comment by lchrusciel** [comment]
Thanks Ben!

**Comment by vzenix** [comment]
Hi.

y try do `composer create-project sylius/sylius-standard my-sylius-shop` It's work, but in bin folder doesn't create "console" and when next time i try do `cd my-sylius-shop` `php bin/console sylius:install` it fails.

PHP version: 7.0.10
System: Windows 7

sylius_problem

**Comment by pamil** [comment]
@vzenix can you also include the output of `composer create-project`?

**Comment by pamil** [comment]
Btw. make sure you're using PHP 7.1+

**Comment by vzenix** [comment]
@pamil Thanks. i change PHP 7.0 to php 7.1 and it's work, next error is an overflow of memory (PHP memory_limit=128M), i change to -1 for testing and the install works.

**Pull request merged**